### ISSUE TITLE
Continuous editing

### ISSUE THREAD

**Issue by fabvio** [ali-vilab/Ranni#9]
Hi, thanks for sharing this amazing work.
I'm currently experimenting with continuous editing. In the readme, it is specified that we should be able to modify the boxes by modifying the prompt. Unfortunately, this not looks immediate to me, can you provide more explanation?

Also, do you have an ETA for the release of the more powerful model? Thanks in advance!

**Comment by thss15fyt** [comment]
Thanks for your interests.

1. For modifying boxes, modify the results in `box answer` in the format of [x coordinate of center, y coordinate of center, box width, box height], then press the `refresh` button.
2. For the more powerful version, we are still working on it without a definite releasing date.

**Comment by fabvio** [comment]
Ok, thanks for the fast reply. I see, so there is no way of actually asking the LLM to modify the panel in the released implementation, am I correct?

**Comment by thss15fyt** [comment]
Yes. We do not include the LLM-based panel modification in the current version. See the chatting-based editing in TODO list.
